mkosi/__init__.py

```diff
@@ -4945,11 +4946,11 @@ def run_verb(args: Args, images: Sequence[Config], *, resources: Path) -> None:
 
     if args.verb == Verb.dependencies:
         _, [deps] = parse_config(
-            ["--directory", "", "--repositories", "", "--include=mkosi-tools", "build"],
+            ["--directory", "", "--repositories", "", *args.cmdline, "--include=mkosi-tools", "build"],
```

Comment:
Looking at this from a completions point of view, this is slightly terrifying. :)

Do we need the full power of being able to pass in any mkosi argument or can we make this 
```suggestion
            ["--directory", "", "--repositories", "", *(f"--profile={p}" for p in args.cmdline), "--include=mkosi-tools", "build"],
```
This way one also doesn't need to pass `--profile` and it's just arguments.

Reply:
OK makes sense let me change this

Reply:
Actually, this would also prevent doing something like `mkosi dependencies -- -d debian` to get the dependencies of debian on Fedora. I figure we may want to support that as well.